jdl/models/jdl-application.ts

```diff
@@ -24,12 +24,18 @@ import JDLApplicationConfigurationOption from './jdl-application-configuration-o
 import JDLApplicationConfiguration from './jdl-application-configuration.js';
 import JDLApplicationEntities from './jdl-application-entities.js';
 import JDLOptions from './jdl-options.js';
+import AbstractJDLOption from './abstract-jdl-option.js';
+
+export type JDLJSONApplicationConfiguration = {
```

Comment:
Same confusion about the relevance: shouldn't it be jdl/models/jdl-application.ts